src/nls/it/strings.js

```diff
@@ -648,7 +648,104 @@ define({
+    "DESCRIPTION_CODE_FOLDING_SAVE_FOLD_STATES"      : "Vero ricorda le selezioni Compresse se chiudi o riapri un file od un progetto",
+    "DESCRIPTION_ATTR_HINTS"                         : "Abilita/Disabilitala suggerimenti attributi HTML",
+    "DESCRIPTION_CSS_PROP_HINTS"                     : "Abilita/Disabilitala suggerimenti delle proprietà CSS/LESS/SCSS",
+    "DESCRIPTION_JS_HINTS"                           : "Abilita/Disabilitala suggerimenti del codice JavaScript",
+    "DESCRIPTION_PREF_HINTS"                         : "Abilita/Disabilitala suggerimenti del codice delle Preferenze",
+    "DESCRIPTION_SPECIAL_CHAR_HINTS"                 : "Abilita/Disabilitala suggerimenti entità HTML",
+    "DESCRIPTION_SVG_HINTS"                          : "Abilita/Disabilitala suggerimenti di codice SVG",
+    "DESCRIPTION_HTML_TAG_HINTS"                     : "Abilita/Disabilitala suggerimenti di tag HTML",
+    "DESCRIPTION_URL_CODE_HINTS"                     : "Abilita/Disabilitala suggerimenti URL in HTML & CSS/LESS/SCSS",
+    "DESCRIPTION_DRAG_DROP_TEXT"                     : "Abilita/Disabilitala funzionalità del Drag & Drop",
+    "DESCRIPTION_HEALTH_DATA_TRACKING"               : "Abilita il tracciamento di Health Data",
+    "DESCRIPTION_HIGHLIGHT_MATCHES"                  : "Abilita l'evidenziazione automatica delle stringhe corrispondenti in tutto il documento",
+    "DESCRIPTION_HIGHLIGHT_MATCHES_SHOW_TOKEN"       : "Evidenziare tutte le stringhe che corrispondono il token il cursore si trova attualmente in (nessuna selezione necessaria)",
+    "DESCRIPTION_HIGHLIGHT_MATCHES_WORDS_ONLY"       : "Evidenziare solo quando la selezione è un token completo",
+    "DESCRIPTION_INSERT_HINT_ON_TAB"                 : "Vero per inserire il codice suggerimento attualmente selezionato nella scheda",
+    "DESCRIPTION_NO_HINTS_ON_DOT"                    : "Vero per non mostrare automaticamente JS suggerimenti sul codice quando . viene digitato",
+    "DESCRIPTION_JSLINT_OPTIONS"                     : "Un oggetto con le opzioni predefinite per JSLint",
+    "DESCRIPTION_JSLINT_OPTIONS_ASS"                 : "Vero per consentire espressioni di assegnamento",
+    "DESCRIPTION_JSLINT_OPTIONS_BITWISE"             : "Vero per consentire agli operatori bit a bit",
+    "DESCRIPTION_JSLINT_OPTIONS_BROWSER"             : "Vero se globali del browser standard, devono essere predefinite",
+    "DESCRIPTION_JSLINT_OPTIONS_CLOSURE"             : "Vero per consentire idiomi Google Closure",
+    "DESCRIPTION_JSLINT_OPTIONS_CONTINUE"            : "Vero per consentire istruzione continue",
+    "DESCRIPTION_JSLINT_OPTIONS_COUCH"               : "Vero se CouchDB globali devono essere predefiniti",
+    "DESCRIPTION_JSLINT_OPTIONS_DEBUG"               : "Vero per consentire dichiarazioni debugger",
+    "DESCRIPTION_JSLINT_OPTIONS_DEVEL"               : "Vero se globali del browser utili in fase di sviluppo devono essere predefiniti",
```

Comment:
**`true se le variabili globali dei browser utili in fase di sviluppo`**` devono essere `**`abilitate`**